### DISCUSSION TITLE
summary api

### DISCUSSION THREAD

**Discussion by sinkaroid** [upptime/upptime#375] [Ideas] [answered]
Ty for this beautiful service
This would be great helpful if there is a summary.json in /api dir which contains a quick overall status such as all response time, and all uptime in one file.

**Comment by FxllenCode** [comment, marked as answer]
Does this not exist? There's a summary.json in /history, right?

**Reply by jayantkatia** [reply]
yes, it exists. ```summary.json``` is generated and updated in ```/history```.

**Reply by sinkaroid** [reply]
Whoops ffs my bad, yes it is
